Article: 아티클 생성 API 구현

wowddok99 opened the pull request:
## Issues

- Resolves #98
- Resolves #99
- Resolves #100
- Resolves #101
- Resolves #102
 
## Description

- #97
  - 아티클 모듈 내부 구조 생성(Issue 98~102 참고)
  - 아티클 생성 API 구현
  
<br />

## Review Points
- 작업량이 많아 리뷰하기 힘드실것 같아서 팀원분들과 중간 중간 소통하면서 작업을 진행하였습니다.
- 눈에 띄는 위주로만 봐주시면 될듯합니다.

<br />

## How Has This Been Tested?
- 포스트맨으로 실행해서 육안으로 확인하였습니다.

<br />

wowddok99 committed:
feat(article): set up article structure and implement create API

silberbullet commented on services/article/driving/web-mvc/src/main/java/nettee/article/driving/web/ArticleCommandApi.java:
```diff
@@ -17,6 +18,9 @@
+
     @Operation(summary = "아티클 생성", description = "블로그 ID에 해당하는 아티클을 생성합니다.")
     @ApiResponses(value = {
             @ApiResponse(responseCode = "201", description = "성공")
```
다음부터는 @ApiResponses는 빼도 됩니다. Swagger @HTTPSTATUS_OK를 자동으로 읽어서 성공 일때 reponseCode를 제공합니다.

<img width="266" height="214" alt="image" src="https://github.com/user-attachments/assets/a8f7d4cd-bfdd-48d3-986a-f5b80505061d" />

silberbullet commented on services/article/application/src/main/java/nettee/article/service/ArticleCommandService.java:
```diff
@@ -0,0 +1,19 @@
+    public Article createArticle(Article article) {
+        return articleCommandPort.save(article);
+    }
+}
```
다음 부터는 클래스에 `{` 열거 이후에는 공백을 만들어 주시고 변수와 @Overide도 공백 넣어주세요!

wowddok99 replied:
말씀해주신 부분 수정해서 커밋하였습니다!
꼼꼼한 리뷰 감사드립니다😁

wowddok99 committed:
refactor(jpa): revert to previous state by removing unnecessary annotations

restores original logic before annotation changes

silberbullet reviewed (approved):
### 고생하셨습니다 👍 

정상적인 아티클 도메인 생성을 포스트맨을 통해 육안으로 확인 하였습니다.

다만 코드적으로 언급한 부분을 참고해주시고 다음 PR에도 반영해주시면 감사합니다!

wowddok99 committed:
refactor(article): remove `@ApiResponses` and fix single blank line conventions

removed unnecessary annotations and ensured consistent use of single blank lines
